src/main/menu/application-menu-items.injectable.ts

```diff
@@ -69,7 +69,8 @@ const applicationMenuItemsInjectable = getInjectable({
             label: `About ${productName}`,
             id: "about",
             click(menuItem: MenuItem, browserWindow: BrowserWindow) {
-              showAbout(browserWindow);
+              showAbout(browserWindow)
+                .catch(error => logger.error(`[MENU]: Failed to show Lens About view`, { error }));
```

Comment:
It would be better to only suppress specific errors, instead of all of them. The specific error may not be fatal, but actual fatal errors should be rethrown.